### PR TITLE
Added documentation for media_player.cambridgeaudio

### PR DESCRIPTION
**Pull request by sebk-666**
**Description:**
Added documentation for the cambridgeaudio platform (https://github.com/home-assistant/home-assistant/pull/14289).

### CODE REVIEW

**Comment by sebk-666** [comment]
Does anyone see what's wrong with the document?
Netlify complains about something in line 12 which is within the header section.
Am I even supposed to add the header section or is this generated automatically somehow?
I just took another media_player markdown file and modified it...

Sebastian

PS: Oh, ignore that - I just had a closer look at the error message and saw that it does not refer to my file.

File: source/_components/media_player.cambridgeaudio.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+description: "Instructions on how to integrate Cambridge Audio network audio players (using their StreamMagic platform) into Home Assistant."
+date: 2018-05-10 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: cambridgeaudio.png
+ha_category: Media Player
+ha_iot_class: "Local Polling"
+ha_release: 0.68
```
Comment (frenck) [inline source/_components/media_player.cambridgeaudio.markdown]: Home Assistant `v0.68` has already been shipped. Please update this to the next major version, which is currently `v0.73`.

File: source/_components/media_player.cambridgeaudio.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## {% linkable_title Configuration %}
+
+The platform will be loaded automatically by discovery component. If you want to configure it manually, add the following to your `configuration.yaml` file:
+
+```yaml
+media_player:
+  - platform: cambridgeaudio
+    name: CA851N
+    host: 192.168.1.31
+    poweroff_command: OFF
```
Comment (frenck) [inline source/_components/media_player.cambridgeaudio.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]
Thank for for sending in this PR @sebk-666!
I've reviewed the documentation and left you some comments. Could you please take a look? 👍

File: source/_components/media_player.cambridgeaudio.markdown (hunk 2 above)
Comment (sebk-666) [reply source/_components/media_player.cambridgeaudio.markdown]: Thanks for reviewing! I've made the requested changes.

**Comment by frenck** [comment]
Thank you @sebk-666 for updating the PR! 🥇 
I've made a couple of small tweaks as well.

✅  Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — APPROVED** [review]

**Pull request closed without merging**